docs/docs/monitoring/analytics/getting-started-with-analytics.mdx

```diff
@@ -213,6 +220,40 @@ analytics but lacks the necessary write performance to directly stream data to i
 
 :::
 
+### BigQuery
+
+These instructions assume you have running a PostgreSQL deployment in [CloudSQL](https://cloud.google.com/sql/docs/postgres).
+You can however use [Datastream for BigQuery](https://cloud.google.com/datastream/docs) to stream data from PostgreSQL
+hosted anywhere on any cloud.
+
+[Prerequisites](https://cloud.google.com/datastream/docs/before-you-begin) to using DataStream include:
```

Comment:
what feels weird to me is the order of sentences up until this one:
- line 225: these instructions are for Postgres in Cloud SQL
- line 226: but you can use datastream if you use postgres outside of GCP
- line 229: to use datastrea, etc...

Do users also need DataStream for CloudSQL postgres?